UHResidentInformationAPI/V1/Gateways/UHGateway.cs

```diff
@@ -107,7 +107,7 @@ public List<ResidentInformation> GetAllResidents(string houseReference = null, s
             //Left Join on tagRef to get contactNo from cccontactLink
             var residentWithContacts = residentWithTagRefs
             .GroupJoin(
-                    _uHContext.ContactLinks,
+                    _uHContext.ContactLinks.Where(c => c.PersonNo != null && c.TagRef != null),
```

Comment:
Is this equivalent to doing an inner join?

Reply:
This should be the equivalent of an outer join.
This should give us records that match with the composite key.